dcicutils/s3_utils.py

```diff
@@ -260,52 +271,73 @@ def health_page_get(self, health_page_key: str, *, default: Any = None, force: b
 
     ACCESS_KEYS_S3_KEY = 'access_key_admin'
 
-    def get_access_keys(self, name=ACCESS_KEYS_S3_KEY):
+    def get_access_keys(self, name: str = ACCESS_KEYS_S3_KEY) -> ServerAuthDict:
+        # TODO: Make .get_access_keys() do better error checking?
+        #       I have marked the return value as ServerAuthDict because it's CLEAR that various callers
+        #       assume that values coming back from this function are in the right form. e.g.,
+        #       ff_utils.get_authentication_with_server will give a hard error if this ever returns
+        #       something with a server missing.  So presumably we mostly do the right thing here.
+        #       It would be much better to properly error-check the result here or even in .get_key().
+        #       -kmp 1-Mar-2023
```

Comment:
If you want to error check do it here instead of in `get_key` which does need to be able to decode/decrypt either JSON or plaintext data.

Reply:
I still kinda think, after doing more research, that the plaintext data is no longer needed and that we just have unit tests that are keeping the old path alive. But not going to obsess about that now. We can circle back for that.